docs/data/material/getting-started/support/support.md

```diff
@@ -61,12 +61,13 @@ The open-source community is always welcome to submit new features and bug fixes
 
 The current status of each MUI version is as follows:
 
-- MUI Core v5 (emotion): ✅ In active development.
-- MUI Core v4 (hooks): ⚠️ Only handle security fixes.
-- MUI Core v3 (change supported browsers): ❌ Inactive.
+- MUI Core v5: **✅ In active development**.
+- [MUI Core v4](https://v4.mui.com/): **⚠️ Only handle security fixes**.
+  - Regarding the security fixes, v4 will remain on long-term support (i.e security updates and fixes for regressions caused by Chrome, React, etc.) until the next major, v6, is fully released, at which time all support for it will end.
```

Comment:
Would it make sense to consolidate this content inside https://mui.com/versions/? I wonder because we are likely to update the two pages at the same time when we work on the versions. In practice, it would mean reversing the content hosting in https://mui.com/versions/#support-policy. But it's unclear if it's better, a random thought.

---

On a different, more serious note, the LTS seems important enough to have a dedicated header about it. To anchor that it's a thing, not an on-time v4 thing. 

```suggestion
- [MUI Core v4](https://v4.mui.com/): **⚠️ Only handle security and regressions fixes**. See the [LTS docs](xxx/#lts) for more details.
```

Reply:
+1 for a dedicated header.